src/Components/SelectableCard/SelectableCard.tsx

```diff
@@ -117,8 +122,9 @@ export const SelectableCard = forwardRef<
             value={label}
             defaultChecked={selected}
             disabled={disabled}
-            tabIndex={-1}
+            {...(tabIndex ? {tabIndex: -1} : {})}
```

Comment:
It looks like previously, the tab index was always -1 as to take this out of the contention for keyboard navigation. 

If I'm reading this operation correctly (which I could be totally messing up lol ), with this new change, if no tab index is passed in, tab index won't be defined for this input, meaning it can be keyboard navigated?

Reply:
This seems like could be a potential breaking change